packages/core-mobile/app/navigation/WalletScreenStack/WalletScreenStack.tsx

```diff
@@ -240,177 +241,180 @@ function WalletScreenStack(props: Props): JSX.Element {
     <BridgeProvider>
       <NFTMetadataProvider>
         {isTestnet && <TestnetBanner />}
```

Comment:
<img width="1304" alt="Screenshot 2024-09-10 at 9 42 21 AM" src="https://github.com/user-attachments/assets/0e78d63b-3b3e-4259-a880-e738097fdc74">

Reply:
This is something I fixed before opening this new PR and I can't reproduce at all anymore. Can you give me some info on the device you're using? That could help me pinpoint the problem

Reply:
I was testing on my Samsung S22+